Task5/index.html

```diff
@@ -2,134 +2,149 @@
+        <link rel="stylesheet" type="text/css" href="css/main/tour/old-price.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/price.css">
     </head>
     <body>
         <header >
             <div class="content header__content">
                 <div class="header__menu-logo">
-                    <img src="images/меню.png">
+                    <img src="images/menu.png">
                 </div>
-                <ul class="header__menu col-menu">
-                    <li class="menu__item menu__user-icon selected-item text">Личный кабинет</li>
-                    <li class="menu__item menu__currency-icon selected-item text">Рубль</li>
-                    <li class="menu__item selected-item text">Ru</li>
-                    <li class="menu__item text"> <strong>8 500-61-05</strong> </br>
+                <ul class="header__menu header__inline-menu">
+                    <li class="menu__item menu__user-icon template_selected-item template__text">Личный кабинет</li>
+                    <li class="menu__item menu__currency-icon template_selected-item template__text">Рубль</li>
+                    <li class="menu__item template_selected-item template__text">Ru</li>
+                    <li class="menu__item template__text"> <span class="template__strong-text">8 500-61-05</span> </br>
                         Круглосуточная поддержка</p>
                 </ul>
                 <div class="header__label">
                     <img class="header__logo" src="images/logo.png">
-                    <p class="header__headline line1 text">Морские прогулки</p>
-                    <p class="header__headline line2 text">ПО ЧЁРНОМУ МОРЮ</p>
-                    <button class="simple-button header__button">Выбрать круиз</button>
+                    <p class="header__headline line1 template__text">Морские прогулки</p>
+                    <p class="header__headline line2 template__text">ПО ЧЁРНОМУ МОРЮ</p>
+                    <button class="template__simple-button header__button">Выбрать круиз</button>
                 </div>
             </div>           
-            <img class="header__background-image">
         </header>
         <main>
             <div class="content">
                 <div>
                     <p class="text main__catalog-text">Каталог круизов</p>
-                    <p class="text main__sort-text selected-item">Сортировать круизы по <em>цене</em></p>
+                    <p class="text main__sort-text template_selected-item">Сортировать круизы по <em>цене</em></p>
                 </div>
                 <div class="grid tours">
-                    <div class="main__tour tour1 bestseller-label">
-                        <img class="tour__image" src="images/tour1-image-lg.png">
+                    <div class="main__tour tour1 template__bestseller-label">
+                        <img class="tour__image" srcset="images/tour1-image-sm.png 320w,
```

Comment:
You have to add `sizes` attribute as well to responsive images.

Reply:
Added